lform.ts

```diff
@@ -220,3 +242,20 @@ export interface DatePeriod {
   start?: string;
   end?: string;
 }
+
+export interface CssExtensions {
+  extension?: [{
+    url?: string;
+    valueString?: string;
+  }];
+}
+
+export interface codeListOptions {
+  code?: string;
+  display?: string;
+  system?: string;
+}
+
+export interface columnHeadOption {
```

Comment:
Please fix capitalization - make ColumnHeadOption not columnHeadOption.

Reply:
Added the changes in the Interface